README.md

```diff
@@ -0,0 +1,88 @@
+[![Contributors][contributors-shield]][contributors-url]
```

Comment:
I would align these with https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#badges

And add a comment link with that.